lib/connectors_sdk/share_point/http_call_wrapper.rb

```diff
@@ -20,6 +20,47 @@ def name
         'SharePoint'
       end
 
+      def extractor(params)
```

Comment:
Is this `extractor` and `extract` method override needed from what's already in https://github.com/elastic/connectors/blob/main/lib/connectors_sdk/base/http_call_wrapper.rb?

Reply:
Ohhh it was a merge conflict for me, fixed in 315ae4a887529696745af50556e6a846433e64ff